firebase/google-services.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
Please git cp this file to preserve history.

Reply:
I am not sure how to do this. I see that 'git cp' isn't a command and I tried following both [these](https://devblogs.microsoft.com/oldnewthing/20190919-00/?p=102904) and [these](https://stackoverflow.com/questions/16937359/git-copy-file-preserving-history) instructions without any success. How do you suggest I move forward?

Reply:
Ah, I was thinking of git mv not copy. Is it possible for us to move the current file here rather than duplicating it? I'm concerned about it becoming out of date.

Reply:
I will try that but I recall there being an issue with gradle building if google-services.json wasn't also in app.